### PR TITLE
Adding README file.

### PR DESCRIPTION
**Pull request by mithro**
README is based on @suzyh's notes on how to get running.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+Install [Bower](http://bower.io/#install-bower)
+```shell
+npm install -g bower
+```
+
+Install [Polymer](https://www.polymer-project.org/1.0/docs/start/getting-the-code.html)
+```shell
+bower install --save Polymer/polymer
```
Comment (dstoc) [inline README.md]: I think you just want to install the bower dependencies with `bower install`

`bower install --save Polymer/polymer` will install the latest version of polymer and then update the `bower.json` file to reference that (but any other dependencies would not be installed).

Comment (mithro) [inline README.md]: Done.

File: bower.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "tests"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.2.0",
+    "polymer": "Polymer/polymer#^1.4.0",
```
Comment (shans) [inline bower.json]: Don't change this dependency as part of this pull request.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+## Deploying
+
+If you have access to the chromez app itself:
+
+ * To push changes to chromez app, from animations-chromez directory
+   > google_appengine/appcfg.py update .
+
+ * Consoles
+   - ["chromez-app" project](http://console.developers.google.com/)
+   - ["dashboard"](https://pantheon.corp.google.com/home/dashboard?project=chromez-app)
```
Comment (shans) [inline README.md]: I don't think we want this link here...

Comment (suzyh) [inline README.md]: Uh, yeah. These instructions started out as my notes to myself. I didn't expect them to end up in a public README.

**Comment by shans** [comment]
I think to make this mergable we'd need to:
- Add a new section at the top that points to chromez-app.appspot.com and talks about how you can load configs from arbitrary URLs.
- Add a section describing the config format.
- Move all of the 'running, etc' stuff into a sub-section further down the page under a heading 'if you want to hack on chromez' (or similar)
- Remove the 'deploying' section entirely.

I understand this is a fair amount of work. If you decide to just close this PR that's fine - we can sort something out later.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+ 3) Type into the box the configuration such as `configs/animations-ave.json` or `configs/blink.json`
+
+## Contributing
+
+Make changes, commit, push to origin, create pull request to
+[shans/chromez](https://github.com/shans/chromez) and get it merged.
+
+## Deploying
+
+If you have access to the chromez app itself:
```
Comment (alancutter) [inline README.md]: This needs to include downloading the third party libraries: `pip install -t lib google-api-python-client`

Comment (mithro) [inline README.md]: Done.

**Comment by mithro** [comment]
Added a section at top for you to fill in with the details when you get around to it.
Move the stuff under a "Developer Instructions".
Fixed the deploying instructions by linking to the external url.

File: bower.json (hunk 2 above)
Comment (mithro) [inline bower.json]: Done.

File: README.md (hunk 3 above)
Comment (mithro) [inline README.md]: I changed the link to the external equivalent and rewrote history to remove it.

**Comment by shans** [comment]
I don't want us to have deploy instructions up at all. Only a tiny handful of people will ever be allowed to deploy to chromez-app.

**Comment by mithro** [comment]
Ask discussed at your desk, I have made the deploy instructions generic for deploying to your own App Engine instance.

**Comment by shans** [comment]
Now they're just generic deploy instructions for how to deploy any app-engine app. There's already documentation for this. Just remove the deploy section please.

**Comment by alancutter** [comment]
It's nice to have the google-api-python-client step covered, we got caught out on that one earlier as we didn't know about the dependency.

It'd be good to mention running `bower install` before uploading as well in case the bower dependencies have changed (again speaking from experience).

**Comment by shans** [comment]
Again, I'm perfectly happy to have specific instructions for chromez-app in a private doc that is shared among the maintainers (i.e. Doug, Suzy, I). They don't need to be here.

**Comment by mithro** [comment]
I've removed the deploy instructions as requested.

**Comment by shans** [comment]
Thanks!

**Pull request merged**